src/errors.ts

```diff
@@ -29,13 +30,34 @@ class InputError extends Error {
  */
 class EnvironmentError extends Error {
   /**
-   * Path of environment attribute that originated the error.
+   * Path of context attribute that originated the error.
    */
   path: string[]
 
   constructor(message: string, path: string[] = []) {
     super(message)
-    this.name = 'EnvironmentError'
+    this.name = 'ContextError'
```

Comment:
Maybe this one should keep being:
```suggestion
    this.name = 'EnvironmentError'
```

Reply:
I think it might indeed be easier to keep the previous `EnvironmentError` name until we release the breaking change, just in case there is someone expecting this name on the serialized data and accessing it directly.